lsp-protocol.el

```diff
@@ -260,7 +260,8 @@ Allowed params: %s" interface (reverse (-map #'cl-first params)))
       (defalias 'lsp-copy 'copy-sequence)
       (defun lsp-member? (from key)
         (when (listp from)
-          (plist-member from key))))
+          (plist-member from key)))
+      (defalias 'lsp-structure-p 'json-plist-p))
```

Comment:
Is using `plistp` enough? Or you may have to include `(require 'json)` for this.

Reply:
AFAIK, `plistp` does not exist. So I will include the require.

Reply:
It's in the latest Emacs snapshot :)
https://github.com/emacs-mirror/emacs/commit/dedd19a2f5cbf16c8ac8a122b0c39ee4e178b9e8
Anyway, using `json-plist-p` is fine.